client/jetpack-onboarding/steps/homepage.jsx

```diff
@@ -33,16 +46,22 @@ class JetpackOnboardingHomepageStep extends React.PureComponent {
 							'We can pull the latest information into your homepage for you.'
 						) }
 						image={ '/calypso/images/illustrations/homepage-news.svg' }
+						href={ forwardUrl }
+						onClick={ this.handleHomepageSelection( 'posts' ) }
```

Comment:
Is this patch also adding `Recent news and updates`? (entire `/homepage` data saving).

Reply:
`Recent news and updates` is the default option in WP, and it doesn't require creating any pages, it takes advantage of the homepage option that WP comes up with by default. We need to create pages only when the "static home page" option is selected.